runtime-modules/storage/src/data_directory.rs

```diff
@@ -162,38 +174,55 @@ pub struct DataObjectInternal<
     pub ipfs_content_id: Vec<u8>,
 }
 
+#[derive(Clone, Copy)]
+pub struct Delta {
```

Comment:
Can we change this name to `Voucher`, I think quota is not actually fully accurate.

Reply:
https://github.com/Joystream/joystream/pull/2122/commits/66937035d6a16bbb5985e533c1064139fb8b41c9